network-api/networkapi/mozfest/templates/mozfest/404.html

```diff
@@ -1,8 +1,7 @@
 {% extends "mozfest/mozfest-base.html" %}
 
 {% block socialMetadata %}
-<meta property='og:type' content='website'>
-<meta property='og:locale' content='en_US'>
+{% include "fragments/og_metadata.html" %}
```

Comment:
Would it be better to pull this in at base.html and override in any templates that _don't_ need it?

Reply:
hm, maybe, although that feels like a followup rather than a refactor of what's in place now.